crates/mipsy_utils/src/expand.rs

```diff
@@ -0,0 +1,51 @@
+use std::{borrow::Cow, path::Path};
+use dirs::home_dir;
+
+#[cfg(unix)]
+use users::os::unix::UserExt;
+#[cfg(bsd)]
+use users::os::bsd::UserExt;
+
+#[cfg(windows)]
```

Comment:
nit: this isn't windows specific behaviour, so `#[cfg(windows)]` is unnecessarily restrictive.
this should probably be `#[cfg(not(any(unix, bsd)))]`
iirc the only other target family is `wasm`, so my guess is mipsy_web wouldn't be able to build with this change

Reply:
hmm looks like `bsd` isn't even a target family, I'll just do `cfg(unix)` and `cfg(not(unix))`

Reply:
mipsy_web looks to be building and operating fine